meteor/client/ui/SegmentTimeline/Renderers/VTSourceRenderer.tsx

```diff
@@ -493,8 +494,7 @@ export class VTSourceRendererBase extends CustomLayerItemRenderer<IProps & WithT
 		) {
 			let endOfContentAt: number = vtContent.sourceDuration + (vtContent.postrollDuration || 0)
 
-			const TBD_COUNTDOWN_TO_FREEZE = false // TODOSYNC: To be implemented by tv2, a setting that enables this
-			if (TBD_COUNTDOWN_TO_FREEZE) {
+			if (Settings.useCountdownToFreezeFrame) {
```

Comment:
Is there a PR against NRK with this contribution?

Reply:
Not at the moment, no.